src/components/atoms/utilityComponents/ScrollToHashElement.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useRef } from 'react'
+import { useLocation } from 'react-router-dom'
+
+/*
+* Modified version of
+* https://dev.to/mindactuate/scroll-to-anchor-element-with-react-router-v6-38op
+*/
+
+export function ScrollToHasElement(): React.ReactNode {
+  const location = useLocation()
+  const lastHash = useRef('')
+
+  useEffect(() => {
+    if (location.hash) {
+      lastHash.current = location.hash.slice(1)
+    } else {
+      window.scrollTo(0, 0)
```

Comment:
react-router-dom does not by default scroll to the top of the page when changing routes.